### PR TITLE
add request size limit

### PR DESCRIPTION
**Pull request by Sekir0**
fix security issue
add validation for available file formats

### CODE REVIEW

File: GhostNetwork.Gateway.Api/Users/UsersController.cs [hunk 1]
```diff
@@ -71,12 +71,23 @@ public UsersController(IUsersStorage usersStorage, ICurrentUserProvider currentU
...
             IFormFile file,
             [FromRoute] Guid userId,
             CancellationToken cancellationToken)
         {
+            if (userId.ToString() != currentUserProvider.UserId)
+            {
+                return Forbid();
+            }
+
+            if (file.FileName[^4..] != ".jpg" && file.FileName[^4..] != ".png")
```
Comment (Boberneprotiv) [inline GhostNetwork.Gateway.Api/Users/UsersController.cs:86]: Have you tested this code with file names less than 4 chars?
There is more native way of extracting file extension 
https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getextension?view=net-6.0

File: GhostNetwork.Gateway.Infrastructure/NewsFeedStorage.cs [hunk 2]
```diff
@@ -206,6 +196,16 @@ private static PublicationComment ToDomain(Comment entity)
                     : null);
         }
 
+        private static PublicationComment ToDomain(Comment entity)
```
Comment (Boberneprotiv) [inline GhostNetwork.Gateway.Infrastructure/NewsFeedStorage.cs:199]: Warning

**Review by Boberneprotiv — CHANGES_REQUESTED** [review]

File: GhostNetwork.Gateway.Infrastructure/NewsFeedStorage.cs (hunk 2 above)
Comment (Sekir0) [reply GhostNetwork.Gateway.Infrastructure/NewsFeedStorage.cs:199]: It is not fixable)
Static members should appear before non-static members
Our static method is private so if I bring him up I'll get another warn "'public' members should come before 'private' members"

Comment (Boberneprotiv) [reply GhostNetwork.Gateway.Infrastructure/NewsFeedStorage.cs:199]: It might be discussed but not as part of this request
You might rise another issue

**Review by Boberneprotiv — APPROVED** [review]

**Pull request merged**